norman_utils_external/json_property_searcher.py

```diff
@@ -0,0 +1,27 @@
+from collections import deque
+
+
+class JsonPropertySearcher:
```

Comment:
Design: write yourself according to Norman standards, and not the basic gpt solution